files/zh-cn/web/css/css_grid_layout/basic_concepts_of_grid_layout/index.md

```diff
@@ -198,26 +198,90 @@ Grid 是一个强大的规范，当与 CSS 的其他部分（如[flexbox](/zh-CN
 }
 ```
 
-{{ EmbedLiveSample('fr 单位', '220', '180') }}
+{{ EmbedLiveSample('The_fr_unit', '220', '180') }}
```

Comment:
The macro call here is correct. Please read the [live sample reference](https://developer.mozilla.org/zh-CN/docs/MDN/Writing_guidelines/Page_structures/Live_samples#%E8%BF%90%E8%A1%8C%E5%AE%9E%E4%BE%8B%E5%AE%8F).

Reply:
Get it, thx so much!